lib/rex/logging/sinks/stdout.rb

```diff
@@ -0,0 +1,44 @@
+# -*- coding: binary -*-
+module Rex
+module Logging
+module Sinks
+
+###
+#
+# This class implements the LogSink interface and backs it against stdout
+###
+class Stdout
```

Comment:
This mimics the existing [Stderr logsink](https://github.com/rapid7/metasploit-framework/blob/1fb7824ab6d94ab1a11431d88bd3c0ebcfb692b8/lib/rex/logging/sinks/stderr.rb)